proto/wtools/abase/l0/l1.test/Long.test.s

```diff
@@ -118,15 +121,15 @@ function isPerformance( test )
    /*
     Average of 10 runs of 1 million iteration of 24 _.long.is variations
     Values below are in seconds
-    ╔═══════════════════╤═════╤═════════╤══════════╤══════════╗
-    ║                   │  is │ isOld ? │ isCompact│ isUnfolded │ isUnfoldedSmartOrder │ isUnfoldedSmarter ║
-    ╟───────────────────┼─────┼─────────┼──────────┼──────────╢
-    ║ **Njs : v10.24.1**│3.662│  1.209  │   1.109  │   ?  ║
-    ╟───────────────────┼─────┼─────────┼──────────┼──────────╢
-    ║ **Njs : v14.17.0**│4.244│  1.263  │   1.156  │   ?  ║
-    ╟───────────────────┼─────┼─────────┼──────────┼──────────╢
-    ║Kos : Njs : v12.9.1│5.053│  2.008  │   1.976  │   1.600  ║
-    ╚═══════════════════╧═════╧═════════╧══════════╧══════════╝
+    ╔═══════════════════╤═════╤═════╤═════════╤══════════╤════════════════════╤═════════════════╗
+    ║                   │  is │isOld│isCompact│isUnfolded│isUnfoldedSmartOrder│isUnfoldedSmarter║
+    ╟───────────────────┼─────┼─────┼─────────┼──────────┼────────────────────┼─────────────────╢
+    ║ **Njs : v10.24.1**│0.598│3.592│  1.211  │  1.137   │      0.823         │    0.628        ║
+    ╟───────────────────┼─────┼─────┼─────────┼──────────┼────────────────────┼─────────────────╢
+    ║ **Njs : v14.17.0**│0.596│4.197│  1.275  │  1.217   │      0.863         │    0.590        ║
+    ╟───────────────────┼─────┼─────┼─────────┼──────────┼────────────────────┼─────────────────╢
+    ║Kos : Njs : v12.9.1│     │     │         │          │                    │                 ║
```

Comment:
![image](https://user-images.githubusercontent.com/7414255/120447025-d3bdb980-c392-11eb-87a1-59ee2a7273c5.png)
Lets make it looking similar

Reply:
![image](https://user-images.githubusercontent.com/6197989/120451645-3cb32a80-c3af-11eb-9eb6-9d6a11ecd9fa.png)

Like this?